server/test/services/program/ProgramServiceTest.java

```diff
@@ -2353,6 +2355,59 @@ public void updateLocalizations_updatesExistingLocale() throws Exception {
                     .build()));
   }
 
+  @Test
+  public void updateLocalizations_blockTranslationsProvided() throws Exception {
```

Comment:
**suggestion, blocking**: can we also add a test that the validation fails correctly?

Reply:
done